cmd/find/grammar.go

```diff
@@ -13,104 +13,98 @@ type atom struct {
 	parsePredicate func(tokens []string) (Predicate, []string, error)
 }
 
-var allAtoms = []*atom{
-	notOp,
-	parensOp,
-	namePrimary,
-	truePrimary,
-	falsePrimary,
-}
-
-// Map of <token> => <atom>. This is initialized in ParsePredicate. Unfortunately,
-// we cannot intialize atoms here because doing so would result in an initialization
-// loop compiler error (e.g. notOp => atoms => notOp).
+// Map of <token> => <atom>.
```

Comment:
I think a comment that this is populated by calls to `newAtom` would be helpful.